js/helper.js

```diff
@@ -105,13 +105,25 @@ function requestUserData(notify, callback) {
       if (notify === true && currentData.refreshInterval != 0){
           var notified = false;
           if (nbReviews > 0 && nbReviews != currentData.nbReviews) {
-            createNotification("You have " + nbReviews +" reviews available.", "https://www.wanikani.com/review", "reviews");
-            notified = true;
+            if (currentData.nbReviews === 1) {
+            // special case of a single review
+              createNotification("You have " + nbReviews +" review available.", "https://www.wanikani.com/review", "reviews");
```

Comment:
[Not exactly](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) ;)